app/models/qdm/tacoma/measure.rb

```diff
@@ -72,8 +72,9 @@ class Measure
     # Relations to other model classes
     belongs_to :user
     belongs_to :bundle, class_name: 'HealthDataStandards::CQM::Bundle'
-    has_and_belongs_to_many :records, inverse_of: nil
+    has_and_belongs_to_many :patients, inverse_of: nil
     has_one :package, class_name: 'CqlMeasurePackage', inverse_of: :measure, dependent: :delete
+    has_many :individual_results
```

Comment:
If there is a relationship of Patient to result, and Patient to measure, do you also need Measure to Results?

Reply:
I considered this. I was trying to think of a scenario where getting all the results for a Measure directly would be more useful than being able to get them through the Patients.

Reply:
Even in Bonnie, it's my understanding that Result access goes through the Patients, rather than through Measures.